src/README.md

```diff
@@ -0,0 +1,7 @@
+# src directory
+
+Everything that will be deployed in production should be in this or the [/public directory](../public/). The latter being the location for the static files.
+
+This does not mean everything in this directory is deployed in production. For example the [Storybook](https://storybook.js.org/) stories and tests are stripped out, but kept here for better Developer eXperience (DX) as colocated files are easier to understand and maintain.
```

Comment:
```suggestion
This does not mean everything in this directory is deployed in production. For example the [Storybook](https://storybook.js.org/) stories and tests are stripped out, but kept here for better Developer experience (DX) as colocated files are easier to understand and maintain.
```

Reply:
That's not a mistake, that's a convention

Reply:
TIL